blackbox/docs/general/builtins/scalar.rst

```diff
@@ -2338,6 +2338,32 @@ Example::
     +--------------------+
     SELECT 1 row in set (... sec)
 
+.. _pg_typeof:
+
+``pg_typeof(any)``
```

Comment:
We generally don't use the synopsis as a section heading, so please use just ``pg_typeof``